etc/examples/tf-model/manifest.yml

```diff
@@ -15,7 +15,7 @@ data_stores:
     training_results:
       container: tf_trained_model
     connection:
-      auth_url: http://s3.default.svc.cluster.local
+      auth_url: test
```

Comment:
Do we have to change the `auth_url` to test? If so, we need to change all the `sed` commands and examples in our READMEs to make it consistent.